libraries/ui-material/Accordion/index.jsx

```diff
@@ -44,80 +46,95 @@ function Accordion(props: Props) {
   const controlsId = testId ? `${testId}-content`.replace(/[^\w\s]/gi, '-').replace(' ', '-') : 'accordion-content';
 
   return (
-    <AccordionContainer open={startOpened}>
-      {({ handleOpen, handleClose, open }) => {
-        const clickHandlers = {
-          onClick: open ? handleClose : handleOpen,
-          onKeyDown: open ? handleClose : handleOpen,
-          role,
-          tabIndex: '0',
-        };
+    <div className="ui-material__accordion-container">
+      <AccordionContainer open={startOpened}>
+        {({ handleOpen, handleClose, open }) => {
+          const clickHandlers = {
+            onClick: open ? handleClose : handleOpen,
+            onKeyDown: open ? handleClose : handleOpen,
+            role,
+            tabIndex: '0',
+          };
 
-        return (
-          <React.Fragment>
-            <div
-              {... (openWithChevron ? {} : clickHandlers)}
-              className={classnames(
-                'ui-material__accordion-title',
-                className,
-                chevronPosition === 'right'
-                  ? styles.toggle.toString()
-                  : styles.toggleLeftAligned.toString()
-              )}
-              data-test-id={testId}
-              key="accordion-toggle"
-              aria-expanded={open}
-              aria-controls={controlsId}
-              aria-label={handleLabel}
-            >
-              {chevronPosition === 'left' ? (
-                <div
-                  className={styles.chevronContainerLeft}
-                  {... (openWithChevron ? clickHandlers : {})}
-                  aria-label={i18n.text(open ? 'favorites.close_list' : 'favorites.open_list')}
-                >
-                  <ChevronIcon
-                    className={open ? styles.chevronOpen : styles.chevronClosed}
-                  />
+          return (
```

Comment:
since the component is already functional and it's quite short, i think that a refactoring (moving the inner code to a custom component) wouldn't create much advantage.